### ISSUE TITLE
Výběr nadřazeného objektu - chybí struktura ve spodní polovině okna

### ISSUE THREAD

**Issue by luckajirku** [proarc/proarc-client#167]
Bylo by možné vrátit u výběru nadřazeného objektu tu spodní polovinu okna se strukturou dokumentu, která se osvědčila ve stávajícím rozhraní? U "Hledat" se to už vrátilo (viz https://github.com/proarc/proarc-client/issues/69), šlo by to i tady?

**Comment by luckajirku** [comment]
A teď už nejde ani proklikat na nižší úroveň (pokud chci strany přesunout přímo do čísla periodika, nikoli pod titul) - zřejmě po úpravě v https://github.com/proarc/proarc-client/issues/170

**Comment by luckajirku** [comment]
Tak proklikat níž už zase jde.

**Comment by albertoh** [comment]
Nepsal jsem, ze to mam opravene, proto ze cekam na celkovou opravu zpusobu hledani a razeni. Viz #170

**Comment by SmejkalovaAnna** [comment]
Další: 

- když přesunu blok stran do jiného objektu, chci, aby mi zůstalo levé okno na původním nadřazeném objektu ( třeba výtisk ) - teď se to přepne do objektu, kam jsem přesouvala

**Comment by SmejkalovaAnna** [comment]
a ještě jeden: 

- Okno Vyberte nařazený objekt pro - když ho během práce použiji poprvé, tak bych chtěla, aby se mi při dalším přesouvání dalších objektů objevila - při použití Vyberte nadřazený objekt - struktura objektů, které jsem použila poprvé..... - teď musím při každém dalším přesouvání stran znovu a znovu naklikat cílový objekt.... 
- ve starém klientovi to takto funguje...

**Comment by albertoh** [comment]
Pridal jsem ten strom. 
Opravil jsem problemy vytekani pri vetsim vyberu.

Podivejte se na to, prosim.

**Comment by SmejkalovaAnna** [comment]
@albertoh Pěkný, díky, to vypadá suprově. Já jsem spokojená. Snad budou i ostatní. Ještě jedna věc, ta poslední, co jsem psala:

- Okno Vyberte nařazený objekt pro - když ho během práce použiji poprvé, tak bych chtěla, aby se mi při dalším přesouvání dalších objektů objevila - při použití Vyberte nadřazený objekt - struktura objektů, které jsem použila poprvé..... - teď musím při každém dalším přesouvání stran znovu a znovu naklikat cílový objekt.... - jde mi o to, aby když potřebuju z jedno objektu rozhodit různý počet stran do různých výtisků, aby si pamatoval při prvním přesouvání kam to dávám, a po druhé by se mi při výběru objektu objevil i ten strom,  kam jsem to dávala v kroku předtím, nějak takto: 
![obrazek](https://user-images.githubusercontent.com/24409660/171788601-d68a30a6-1f56-46fb-93e5-89587de6251e.png)

-     ve starém klientovi to takto funguje...

**Comment by albertoh** [comment]
Je to tam. Zkuste

**Comment by SmejkalovaAnna** [comment]
@albertoh Moooooc hezké! Děkuji, funguje pěkně! bude chtít vyzkoušet i @ZdenkaSera  a @luckajirku ?

**Comment by ZdenkaSera** [comment]
@albertoh Za mne 👍, díky! V horním okně funguje i třídění podle všech sloupců @SmejkalovaAnna @luckajirku

**Comment by luckajirku** [comment]
tlačítko Přesunout - ve chvíli, kdy označím nějaký objekt, se text v něm změní - a nevím proč název objektu ukáže obohacený o další znaky: uvozovky, závorka, název, uvozovky, uvozovky, závorka. (tu změnu z: Přesunout k na Přesunout do jsem už dala do překlepů).

**Comment by albertoh** [comment]
Ano, znaky jsou tam spatne. Chtel jsem oddelit nazev objektu "jen" uvozovky, ale nejak se mi to tam nahromadilo.
Chcete to radsi nechat rovnou nazev vybraneho objektu? Bez jakehokoli znaku?
   a)  Přesunout do 1, 1.1.1945
   b)  Přesunout do "1, 1.1.1945"

**Comment by luckajirku** [comment]
A co dvojtečka a mezera? 
Přesunout do: _objekt_

**Comment by albertoh** [comment]
Jasne, vyberte format jak nejvice vyhovuje. Ja jsem jen chtel jasne znazornit kam se to presouva

**Comment by luckajirku** [comment]
To já taky, za ten _objekt_ dosaďte to, co jste tam měl, nemyslela jsem, aby tam bylo přímo to slovo objekt:-)
Přesunout do: 1, 1.1.1945
Přesunout do: Bulletin

**Comment by luckajirku** [comment]
Ještě je to trochu nepřehledné - ve spodní části chybí lišta s názvy sloupců, navíc oproti obrazovce Hledat jsou sloupce v jiném pořadí.

**Comment by albertoh** [comment]
Je to tam

**Comment by albertoh** [comment]
> Přeci jen - nešlo by tu lištu mezi horní a spodní polovinou udělat pousouvací, jako je to ve stávajícím? U rozřazování do čísel periodika je to trochu nepraktické mít tu horní zbytečně vysokou s výběrem dalších titulů - a dole ze struktury toho periodika, ve kterém člověk pracuje, vidět jen část a muset pořád rolovat...

Pridano

**Comment by luckajirku** [comment]
Super, to bylo rychlé. Jen teda po každém přesunu objektu se tady ta lišta vrací na výchozí pozici - zatímco ve stávajícím zůstává tak, jak si to uživatel nastavil. Šlo by i to nastavení lišty pamatovat?

**Comment by albertoh** [comment]
Uz pamatuje

**Comment by luckajirku** [comment]
Pamatuje, funguje, vyhovuje (aspoň nám teda:-))

**Comment by SykoraLukas** [comment]
Uzavřeno na základě labelu během schůzky 17.6.2022.
